deps/wxWidgets/wxWidgets.cmake

```diff
@@ -1,4 +1,8 @@
-set(_wx_git_tag v3.1.5)
+# set(_wx_git_tag v3.1.5) # Does not link -ldep-name-NOTFOUND (see deps/build/dep_wxWidgets-prefix/src/dep_wxWidgets-build/wx-config --libs all | grep NOTFOUND)
+# set(_wx_git_tag v3.1.4-patched) # Links, but Bambu's slic3r_gui depends on wxWebView methods in >=3.1.5 like SetUserAgent
+# set(_wx_git_tag v3.1.6) # wx includes nanosvg, which creates link conflicts (multiple definition) with slic3r which also uses nanosvg, wx nanosvg can't be disabled
+# set(_wx_git_tag v3.1.7) # wx includes nanosvg, which creates link conflicts ( multiple definition) with slic3r which also uses nanosvg, segfaults if nanosvg disabled
+set(_wx_git_tag v3.2.0) # Starts, but gives unhandled AppConfig::Save() exception
```

Comment:
we have submitted a fix for the wxWidgets compiling under linux before,
it should fixed the '-ldep-name-NOTFOUND' issue for v3.1.5
you can try it and remove the changes of this part
--------------------------------------------------------------------
commit 78db2af59906859868a7d32d5fead7f114766950
Author: gerrit <gerrit@bambulab.com>
Date:   Tue Aug 9 10:13:02 2022 +0800

    support linux build for wxWidgets

    Change-Id: I7d4e84c4e82371c6e688b607ac0658c6319d1216
    (cherry picked from commit 6ff1613c3997628ea932d4956f874c2356da0056)

M       cmake/modules/FindGTK3.cmake
M       deps/wxWidgets/0001-wxWidget-fix.patch
M       src/slic3r/CMakeLists.txt